src/components/Login.js

```diff
@@ -0,0 +1,25 @@
+import {Link } from "react-router-dom";
```

Comment:
@veljomiha @BaneProjects seems maybe the prettier plugin is not active in your VS Code - you can set VS Code to automatically format code on every save - https://stackoverflow.com/a/62532674/2516585

Reply:
Okay , we did.